stock_scheduler_assignation_horizon/models/procurement_group.py

```diff
@@ -38,3 +40,24 @@ def _get_moves_to_assign_domain_assignation_limited(self, domain, company):
                 [("date", "<=", max_date)],
             ]
         )
+
+    def get_global_is_moves_assignation_limited(self):
+        get_param = self.env["ir.config_parameter"].sudo().get_param
+        param = "stock_scheduler_assignation_horizon.stock_horizon_move_assignation"
+        return str2bool(get_param(param, False))
+
+    def get_global_moves_limit_horizon(self):
```

Comment:
```suggestion
    def _get_global_moves_limit_horizon(self):
```
Should be private

Reply:
`_get_assignation_horizon_limit` ? :innocent: